update_exchange_rates.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+"""A script to update the exchange rates for 1 EUR in SEK and 1 USD om SEK.
+
+"""
+
+import argparse
+import yaml
+from couchdb import Server
+import datetime
+from forex_python.converter import CurrencyRates
+
+
+def get_current(db, view):
+    rows = db.view("by_date/{}".format(view), descending=True, limit=1).rows
+    if len(rows) != 0:
+        value = rows[0].value
+        return value
```

Comment:
Since after this `if` it is just a `return` statement, you could refactor your code to have only one `return` statement. This is just a suggestion, nothing crucial :)

```
def get_current(db, view):
    value = None
    rows = db.view("by_date/{}".format(view), descending=True, limit=1).rows
    if len(rows) != 0:
        value = rows[0].value
    return value
```

Reply:
True! I think I prefer this one though since the number of rows are the same and I actually think its easier to understand. But yes, I wouldn't mind your solution either.